R/insertPackage.R

```diff
@@ -130,7 +130,7 @@ identifyPackageType <- function(file) {
     ret <- if (grepl("_.*\\.tar\\..*$", file)) {
         "source"
     } else if (grepl("_.*\\.tgz$", file)) {
-        "mac.binary"
+        "mac.binary.mavericks"
```

Comment:
certainly not. miniCRAN also [hacks around it](https://github.com/RevolutionAnalytics/miniCRAN/blob/master/R/makeRepo.R#L85)